src/parameters/Microphysics2M.jl

```diff
@@ -419,6 +419,11 @@ function RainParticlePDF_SB2006(td::CP.AbstractTOMLDict)
     return RainParticlePDF_SB2006{FT}(; parameters...)
 end
 
+const RainParticlePDF_SB2006_MaybeLimited{FT} = Union{
```

Comment:
I don't think it improves code readability. I would vote to just pass the union when needed

Reply:
Ok, I'll revert.